source/slang/slang-capabilities.capdef

```diff
@@ -372,6 +373,8 @@ alias GL_ARB_sparse_texture_clamp = _GL_ARB_sparse_texture_clamp | spvSparseResi
 alias GL_ARB_texture_gather = _GL_ARB_texture_gather | spvImageGatherExtended | metal;
 alias GL_ARB_texture_query_levels = _GL_ARB_texture_query_levels | spvImageQuery | metal;
 alias GL_ARB_shader_clock = _GL_ARB_shader_clock | spvShaderClockKHR;
+alias GL_ARB_shader_clock64 = _GL_ARB_shader_clock + _GL_ARB_gpu_shader_int64 | spvShaderClockKHR;
```

Comment:
I think we might not need GL_ARB_shader_clock64 anymore, since _GL_ARB_shader_clock includes _GL_ARB_shader_clock64 by looking at their definition.

Just leave __glsl_extension(GL_ARB_gpu_shader_int64) should be fine, right?

Reply:
`_GL_ARB_shader_clock` shouldn't include `_GL_ARB_shader_clock64`.
`_GL_ARB_shader_clock64` should be `_GL_ARB_shader_clock` + `_GL_ARB_gpu_shader_int64`.